package-lock.json

```diff
@@ -3794,7 +3794,8 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true
```

Comment:
Why are there package-lock.json changes if the package.json hasn't been changed? 

There´s no need to add this to this commit. Please discard the changes related to package.lock.json